app/javascript/packs/billboardAfterRenderActions.js

```diff
@@ -42,6 +42,22 @@ export function executeBBScripts(el) {
   }
 }
 
+export function implementSpecialBehavior(element) {
+  if (element.querySelector('.js-billboard').dataset.special === 'delayed') {
```

Comment:
Is it possible for `element.querySelector('.js-billboard')` to return null and break this section? I think it would be a good idea to add safe navigation.

Reply:
It shouldn't be null, but good call, I'll add an extra condition.